src/types/user.ts

```diff
@@ -2,8 +2,8 @@ interface User {
   token: string;
 }
 
-interface AvatarShapeType {
-  avatarShape: 'rectangle' | 'normal' | 'large' | 'small' | 'medium';
+interface AvatarSizeType {
+  avatarSize: 'normal' | 'large' | 'small' | 'medium';
 }
```

Comment:
p2;
```suggestion
type  avatarSize = 'normal' | 'large' | 'small' | 'medium';
```
단일 타입은 type으로 선언하는게 관리하기 더 편할거예요!